dependabot.yml

```diff
@@ -0,0 +1,20 @@
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+version: 2
+updates:
```

Comment:
Groups: Should have group assignment for actions and for each dependency group within our pyproject.toml. That will allow for one PR to be posted with dependency updates for anything associated with that group rather than a single PR for each dependency.